stress/shell/src/main/java/alluxio/stress/cli/Benchmark.java

```diff
@@ -80,6 +80,12 @@
   //  into two different calls instead of relying on the same prepare().
   public abstract void prepare() throws Exception;
 
+  /**
+   * Check if there are multiple task to run.
+   * @return return the type of multiple task, if it is not multiple task, return null
+   */
+  public abstract String checkIfMultipleTask();
```

Comment:
this method feels awkward: right now its sole purpose is to indicate whether a benchmark needs to run for all write types, but it sounds like it can support arbitrary multiple tasks. Besides, the return type is just a string, which just leaves to the caller to interpret what that string means for a task.

What we want to have is basically two things: 

1. make write type configurable for `StressMasterBench` and friends.
2. provide easier access to be able to run all possible write types.

For one, this is achieved with your addition of `hdfsConf.set(PropertyKey.Name.USER_FILE_WRITE_TYPE_DEFAULT, mParameters.mWriteType);` to the benchmarks.

For two, IMO it's better to provide a higher level of abstraction than the `Benchmark` base class, which defines the concept of "task". A task is one set of benchmark parameters that are used to invoke a benchmark. You can then enumerate all possible write types in that abstraction, and invoke a series of benchmark runs each time with a different write type. It's even a better idea since you are also working on improving benchmark output. You can do output postprocessing in the said high level abstraction, e.g. providing nice comparison of the performances of the different write types.